0200-mapping.Rmd

```diff
@@ -0,0 +1,158 @@
+
+
+# Mapping
+
+We use QGIS and Mergin Maps for our fish passage data management and analysis. This section will get you started with our mapping procedures.
+
+## QGIS
+
+* QGIS (Quantum Geographic Information System) is a free, open-source software that allows users to create, edit, visualize, analyze, and publish geospatial information.
+
+* Follow this link to the [QGIS Download Page](https://www.qgis.org/en/site/forusers/download.html) to install the latest version.
+
+* Here is some great info for getting up to speed with the [basics of QGIS](https://courses.spatialthoughts.com/introduction-to-qgis.html)
+
+
+## Mergin Maps
```

Comment:
this is where the info from the emails I forwarded to you about getting an account and setting user name goes and this can be the first section in Data Collection and Field Navigation